feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.ts

```diff
@@ -358,4 +396,58 @@ export class ConfiguratorAttributeBaseComponent {
       !this.isReadOnly(attribute)
     );
   }
+
+  /**
+   * Merges value price data received via @see {ConfiguratorValuePriceChanged} events into the given value, if available.
+   * @param {Configurator.Value} value the value
+   * @returns {Configurator.Value} the new value with pricing
+   */
+  protected mergePriceAndValue(value: Configurator.Value): Configurator.Value {
+    const valueName = value.name;
+    if (valueName && this.valuePrices[valueName]) {
+      value = { ...value, valuePrice: this.valuePrices[valueName] };
+    }
+    return value;
+  }
+
+  /**
+   * Creates a text describing the current attribute that can be used as ARIA label.
```

Comment:
where do you read translation key?

Reply:
Depending on the condition the following method will be called: 
- `getAriaLabelForValueWithPrice` or
- `getAriaLabelForValue`
This methods retrieve a translation key for aria-label, that's why I suggested to rephrase a JS doc, for instance:
> /**
>    * Retrieves a translation key that will be used for aria-label attribute.
>    * This translatio key includes price information as well.
>    * If a total price is available this price will be used,
>    * otherwise it falls back to the value price.
>    * If no price is available, no price information will be included into the translation key.
>    *
>    * @param attribute - the attribute
>    * @param value - the value
>    * @returns - translation key
>    */